cmd/cloud-controller-manager/app/controllermanager.go

```diff
@@ -323,7 +323,10 @@ func StartHTTPServer(ctx context.Context, c *cloudcontrollerconfig.CompletedConf
 	if c.SecureServing != nil {
 		unsecuredMux := genericcontrollermanager.NewBaseHandler(&c.ComponentConfig.Generic.Debugging, healthzHandler)
 
-		unsecuredMux.Handle("/metrics/v2", traceProvider.MetricsHTTPHandler()) // Add metricsv2 endpoint
+		const MetricsPath = "/metrics"
+		unsecuredMux.Unregister(MetricsPath) // Unregisterer handler of legacy registerer
```

Comment:
Can we just not register the metrics path with old register?

Reply:
> Can we just not register the metrics path with old register?

It’s better to stick with the old path for the new metrics since it makes scrape config maintenance easier. Plus, separating the two has already been tested for a while.

> And can you list the OTEL metrics from after merging the metrics from legacy register as a test?

Already tested it in the standalone. Will skip pasting it here as it's a large volume of data.

Reply:
yes, agreed